service/controller/appcatalog/v1/resource/index/update.go

```diff
@@ -0,0 +1,79 @@
+package index
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange interface{}) error {
+	configMap, err := toConfigMap(updateChange)
```

Comment:
You called this `cm` in other places. Let's stick to one approach.

Reply:
Thanks, fixed it.